# Computational Genomics: Applied Comparative Genomics
## Project Presentations

Presentations will be a total of 12 minutes: 10 minutes for the presentation, followed by 2 minutes for questions. We will strictly keep to the schedule to ensure that all groups can present in class! 

## Schedule of Presentations


| Slot | Date  | Start | Team Name                         | Team Members                                           | Project Title                                                                                                                    |
| ---- | ----- | ----- | --------------------------------- | ------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
| 1    | 11/20 | 3:00  | Two single-cells, one big problem | Kevin Meza Landeros, YunZhou Liu                       | Cluster-based single-cell RNA-seq variant detection                                                                              |
| 2    | 11/20 | 3:12  | Team Yuxiang Li                   | Yuxiang Li                                             | Contrastive Learning Approach to Integrate Single-Cell scRNA-seq and scATAC-seq for Mechanistic Understanding of Gene Regulation |
| 3    | 11/20 | 3:24  | Team Roujin An                    | Roujin An                                              | Cell Type-Specific SNP-to-Splicing Variants Mapping Using Deep Learning Models                                                   |
| 4    | 11/20 | 3:36  | Team1D                            | Ben Miller                                             | Comparative Genomic Analysis of NOD and (Simulated) NOR Mouse Genomes to Identify Variants Associated with Type 1 Diabetes       |
| 5    | 11/20 | 3:48  | Genomic Visionaries               | Iason Mihalopoulos, Siam Mohammed                      | AR/VR Visualization of Individual Genomes with AI-driven Insights                                                                |
|      |       |       |                                   |                                                        |                                                                                                                                  |
| 1    | 12/2  | 3:00  | Silent Codebreakers               | Cecelia Zhang, Jiarui Yang                             | Benchmarking Non-Coding Mutation Analysis Schemes on Cancer Genomes                                                              |
| 2    | 12/2  | 3:12  | Team Table                        | Oce Bohra, Zoe Rudnick                                 | The emerging contribution of non-coding mutations in glioblastoma development                                                    |
| 3    | 12/2  | 3:24  | Team Brady                        | Brady Bock                                             | DNN analysis of gut microbiomes to predict colorectal cancer disease state                                                       |
| 4    | 12/2  | 3:36  | Variant Visionaries               | Alexandra Gorham, Christine Park, Natalie Vallejo      | Benchmarking Non-coding Variant Scoring Tools for Cancer Pathogenicity Prediction                                                |
| 5    | 12/2  | 3:48  | Human to Plants                   | Xiaojun Gao, Yujia, Yushan Zou                         | Evaluation of applicability of ChromHMM for Plants in Chromatin States and Gene Expression                                       |
| 6    | 12/2  | 4:00  | Team Miller                       | Logan Miller                                           | Population-Specific Evolutionary Hotspots in Human Genomes                                                                       |
|      |       |       |                                   |                                                        |                                                                                                                                  |
| 1    | 12/4  | 3:00  | SE Palmeiras                      | Caleb Hallinan, Jamie Moore, Rafael dos Santos Peixoto | Evaluating cell-type clustering algorithmâ€™s robustness to technical artifacts via synthetic spatial transcriptomics data         |
| 2    | 12/4  | 3:12  | Nuclencoder                       | Amanda Xu, Angela Yang, Jiamin Li                      | DNA Cryptography: Digital Signatures for Encryption to Facilitate Safe Data Storage                                              |
| 3    | 12/4  | 3:24  | Geoguessr                         | Alex Ostrovsky, Nicole Lauren Brown                    | Investigating geographic and environmental effects on soil metagenomes by correlating GIS data                                   |
| 4    | 12/4  | 3:36  | Quetzalli Tlalli                  | Arshana Welivita, Atticus Colwell                      | Benchmarking Methods for Inferring the Ethnicity of an Individual from Their Genotype                                            |
| 5    | 12/4  | 3:48  | Team Barbour                      | Alexis Barbour                                         | Benchmarking non-coding mutation analysis schemes for evaulating Type 1 Diabetes                                                 |
| 6    | 12/4  | 4:00  | All of Us Team                    | Levon Galstyan, Nitish Aswani, Talia Haller            | Genomic Insights into Sleep Patterns, Disease Outcomes, and Biomarker Associations using the All of Us Dataset                   |

## Recommended outline for your talk (~1 minute per slide):

1. Title Slide: Who are you, title, date
2. Intro 1: Whats the big idea???
3. Intro 2: More specifically, what are you trying to learn?
4. Methods 1: What did you try?
5. Methods 2: What is the key idea?
6. Data 1: What data are you looking at?
7. Data 2: Anything notable about the data?
8. Results 1: What did you see!
9. Results 2: How does it compare to other methods/data/ideas?
10. Discussion 1: What did you learn from this study?
11. Discussion 2: What does this mean for the future?
12. Acknowledgements: Who helped you along the way?
13. Thank you!

I strongly *discourage* you from trying to give a live demo as they are too unpredictable for a short talk. If you have running software you want to show, use a "cooking show" approach, where you have screen shots of the important steps.    
